src/Tests/View/PasswordQuickInput.test.ts

```diff
@@ -16,24 +16,24 @@
 
 import {assert} from 'chai';
 
-import {containsNonAscii} from '../../View/PasswordQuickInput';
+import {containsOnlyPrintableAscii} from '../../View/PasswordQuickInput';
 
 
 suite('View', function() {
   suite('PasswordQuickInput', function() {
-    suite('#containsNonAscii()', function() {
+    suite('#containsOnlyPrintableAscii()', function() {
       test('basics', function() {
         const strWithAlphabets = 'abcdefghijkABCDEFGHIJK';
-        assert.isNotTrue(containsNonAscii(strWithAlphabets));
+        assert.isTrue(containsOnlyPrintableAscii(strWithAlphabets));
 
         const strWithNum = '0123456789';
-        assert.isNotTrue(containsNonAscii(strWithNum));
+        assert.isTrue(containsOnlyPrintableAscii(strWithNum));
 
-        const strWithSpecialChar = '\n\t\r!@#$%^&*(){}[]\\';
-        assert.isNotTrue(containsNonAscii(strWithSpecialChar));
+        const strWithSpecialChar = '!@#$%^&*(){}[]\\';
```

Comment:
I removed \n\t\r from the test.

Reasoning: 

(1) \r \n,
They are CR, LR keys which are 'new line' characters based on each os. 
> CR(\r), LF(\n)](https://m.blog.naver.com/taeil34/221325864981)
:When I hit the 'enter' key, the password quick pick returns, so they are not expected to be included in passwords.
(2) \t
When I hit the 'tab' key, the password quick pick disappears, so it is not expected to be included in passwords.

Reply:
Is there any possibility that the key value will be entered through copy and paste?

Reply:
Oh.. sure. I tried a random password with tab and space, and it worked by pasting in.

Reply:
#### Allowiing \t \n by the rule:

It seems \t \n are allows by the rule even though they are control characters.

https://eslint.org/docs/latest/rules/no-control-regex

> The following elements of regular expression patterns are considered possible errors in typing and are therefore disallowed by this rule:

> Hexadecimal character escapes from \x00 to \x1F.
> Unicode character escapes from \u0000 to \u001F.
> Unicode code point escapes from \u{0} to \u{1F}.
> Unescaped raw characters from U+0000 to U+001F.
> Control escapes such as \t and \n are allowed by this rule.

#### More to consideration

It seems 0x7f is a special character(delete) too. 

https://www.geeksforgeeks.org/control-characters/

Reply:
So let me add \n, \r and \t to the regex and see if they are allowed. The other control characters look not to allow from security's view, IMO.

Reply:
❗ 

I tried with ascii 0x1,0x2 and even 0x18(Cancel) and linux os still accept all those control ascii characters.
```
0x1 Start of Heading
0x2  Start of Text
0x18 Cancel
```

Reply:
I tried with 0xff(latin y) 0x80(euro sign), which are extended ascii. They are accepted as a linux passwd but I couldn't login to vscode remote with those passwords